apps/design-system/src/components/Form/Form.stories.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import type { Meta } from "@storybook/react"
+import {
+  TextArea,
+  Field,
+  Form,
+  Group,
+  Fieldset,
+  IconButton,
+  TextInput,
+  Label,
+  Dropdown,
+  ChipInput,
+} from "@asyncapi/studio-ui"
+import { AddIcon, TrashIcon } from "@asyncapi/studio-ui/icons"
+
+const meta: Meta<typeof Form> = {
+  component: Form,
+  parameters: {
+    layout: "fullscreen"
+  },
+}
+
+export default meta
+
+const singleSelectOptions = [
+  { label: "HTTP", value: "http" },
+  { label: "Kafka", value: "kafka" },
+  { label: "Websocket", value: "websocket" },
+  { label: "AMQP", value: "amqp" },
+  { label: "MQTT", value: "mqtt" },
+  { label: "Google PubSub", value: "googlepubsub" },
+  { label: "IBM MQ", value: "ibmmq" },
+  { label: "NATS", value: "nats" },
+  { label: "Pulsar", value: "pulsar" },
+]
```

Comment:
```suggestion
]
const renderSecurityInput = () => (
  <div className="flex gap-3">
    <Dropdown options={[{ label: "User/Password", value: "user/password" }]} value="user/password" />
    <TextInput placeholder="Type something here..." className="grow" />
    <TrashIcon className="w-6 h-6 text-gray-500" />
  </div>
);
```
Refactored this section.